docs/Commands.md

```diff
@@ -120,6 +125,7 @@ Finds the closest entity with the given component or LDF variable (ignoring play
 * `-s`: Prints the entity's settings and spawner ID.
 * `-p`: Prints the entity's position
 * `-f`: If the entity has a destroyable component, prints whether the entity is smashable and its friendly and enemy faction IDs; if `faction` is specified, adds that faction to the entity.
+* `-cf`: check if the faction of this entity is your enenmy or friend
```

Comment:
enemny
also this could be clarified to entity as this would refer to self when inspect only ever refers something else